### DISCUSSION TITLE
Elastic 7.9.3 - Kibana - "Your query is taking a while"

### DISCUSSION THREAD

**Discussion by krullos** [Security-Onion-Solutions/securityonion#2690] [Unsupported Versions]
Since the update from Elastic 6.8.11 to Elastic 7.9.3 (Security Onion 16.04) the following dialogue box pop-up when running Kibana searches.

Your query is taking a while
With an upgraded license, you can ensure requests have enough time to complete

![Upgrade License](https://user-images.githubusercontent.com/47314874/105245492-d20b0a80-5bd6-11eb-9c6b-bcdf8d93b2c3.JPG)

I am using the default Open-Source version that comes with Security Onion.

Does anyone know what changed from 6.8.11 to 7.9.3 that may have started restricting the length of time searches may take?
What are the ramifications of NOT purchasing an Elastic license? Do we only get partial search results now?

**Comment by weslambert** [comment]
HI @krullos , this messaging comes from the Elastic backend.  There is probably tuning that could be done to improve search times, but simply changing the license wouldn't necessarily improve your performance, and you don't need to change the license to search effectively, although there may be other features that you find beneficial.  It is free to enable Elastic Features/Basic license using `so-features-enable`.
